tests/zombienet/zombienet.sh

```diff
@@ -34,14 +33,21 @@ make_bin_dir() {
 
 fetch_zombienet() {
     if [ "$(nixos-version 2>/dev/null)" == "" ]; then
+        # Don't fetch zombienet if it's already present in the system
+        if which zombienet-$zombienet_version >/dev/null; then
+            cp $(which zombienet-$zombienet_version) "$bin_dir/zombienet"
+            echo "✅ Zombienet $zombienet_version installed"
+            return
+        fi
+
         if [ ! -f "$bin_dir/zombienet" ]; then
```

Comment:
We already do this here no?

Reply:
Not quite, each time this runs on a runner that havent fetched zombienet, or on a runner that $bin_dir was cleared we will have to refetch it. Github runner is usually creating some tmp dirs, which makes this action refetch zombienet each time.

What had changed in gh runner is that in /usr/bin/ we will have binaries like:
- zombienet-v1.4.40
- zombienet-v1.4.23
- polkadot-release-v0.9.37
- etc

So our script doesn't have to fetch repos, build anything as it's already present.